test/babashka/neil/test_util.clj

```diff
@@ -0,0 +1,39 @@
+(ns babashka.neil.test-util
+  (:require [babashka.fs :as fs]
+            [babashka.neil :as neil-main]
+            [babashka.process :refer [sh] :as process]
+            [babashka.tasks :as tasks]
+            [clojure.edn :as edn]
+            [taoensso.timbre :as log])
+  (:import (java.io StringWriter)))
+
+(def test-dir (str (fs/file (fs/temp-dir) "neil")))
+
+(defn test-file [name]
+  (doto (fs/file test-dir name)
+    (-> fs/parent (fs/create-dirs))
+    (fs/delete-on-exit)))
+
+(defn ensure-git-repo []
+  (let [opts {:dir test-dir :err :inherit}]
+    (sh "git init -b main" opts)
+    (sh "git config user.name 'Neil Tests'" opts)
+    (sh "git config user.email '<>'" opts)
+    (sh "git add ." opts)
+    (sh "git commit -m 'First commit'" opts)))
+
+(defn test-logging-config [log-stream]
+  {:appenders
+   {:log-stream {:enabled? true
+                 :fn #(binding [*out* log-stream]
+                        (println (force (:output_ %))))}}})
+
+(defn neil [cli-args & {:keys [out] :or {out :edn}}]
+  (let [deps-file (str (test-file "deps.edn"))
+        cli-args' (concat (process/tokenize cli-args) [:deps-file deps-file])
+        log-stream (StringWriter.)]
+    (binding [log/*config* (merge log/*config* (test-logging-config log-stream))
```

Comment:
I added this so we ccan use Timbre in dev and tests without breaking tests that check `:out`. This was useful when I ran into a test failure today that was only showing up on GitHub Actions. It can be removed if there's any problem with it.